Codes/Print.java

```diff
@@ -7,40 +7,54 @@
 import features.*;
 
 class Print {
-    public void print(String s) {
+    public void print(String s1) {
         
         //+ ====> Concatenation
-        //'' ===> String in print function
-        Stack<Character> mainStack = new Stack<Character>();
-        String s1 = "";
-        String s2 = "";
-        int i=6;
+		//'' ===> String in print function
+		String s=s1.substring(6,s1.size()-3);
```

Comment:
Print.java:114: error: cannot find symbol
                String s=s1.substring(6,s1.size()-3);
                                          ^
  symbol:   method size()
  location: variable s1 of type String